<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddNewVehicle</class>
 <widget class="QDialog" name="AddNewVehicle">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>752</width>
    <height>576</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="lblBanner">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>721</width>
     <height>441</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Vehicle Info</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Type</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="typeCombo"/>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Make</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="makeCombo"/>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Model</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="modelCombo"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Year</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="yearCombo"/>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>VIN</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="vinLineEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Base Fare</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="baseFare"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Color</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="colorCombo"/>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>WheelDrive</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="wdCombo"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Passenger Capacity</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="passengerCapacityCombo"/>
      </item>
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Mileage</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="mileageLineEdit">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="QPushButton" name="addBtn">
        <property name="styleSheet">
         <string notr="true">QPushButton#addBtn {
background-color: #6246ff; 
  border: none;
  color: white;
  padding: 15px 32px;
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 16px;
  transition-duration: 0.4s;
}
QPushButton#addBtn:hover {
  background-color: #215a91; /* Green */
  color: white;
}</string>
        </property>
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="closeBtn">
        <property name="styleSheet">
         <string notr="true">QPushButton#closeBtn {
background-color: #6246ff; 
  border: none;
  color: white;
  padding: 15px 32px;
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 16px;
  transition-duration: 0.4s;
}
QPushButton#closeBtn:hover {
  background-color: #215a91; /* Green */
  color: white;
}</string>
        </property>
        <property name="text">
         <string>Close</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
